.github/pull_request_template.md

```diff
@@ -33,8 +33,9 @@ What does the PR accomplish, why was it needed?
 
 ###### Does this affect the toolchain?  <!-- REQUIRED -->
 <!-- Any packages which are included in the toolchain should be carefully considered. Make sure the toolchain builds with these changes if so. -->
-**YES**
```

Comment:
I'm a fan of the changes in this file, but can we get these in a separate PR?

Reply:
That makes a lot of sense.  The origin of these changes came from Daniel McIlvaney, who did a little bit of work on this branch last fall and included these updates in it.  Let me see if I'm able to pull those out for a separate PR, keeping his git info and Sign off.

Reply:
If I've done all the git cleanup correctly, I believe I have now reverted this chunk of changes on this branch and opened a new PR to add them in here: https://github.com/microsoft/CBL-Mariner/pull/963